tests/resources/functional/snowflake/core_engine/functions/conversion/test_to_time_2.sql

```diff
@@ -3,4 +3,4 @@
 SELECT TO_TIME('2018-05-15 00:01:02');
 
 -- databricks sql:
-SELECT TO_TIMESTAMP('2018-05-15 00:01:02', 'yyyy-MM-dd HH:mm:ss');
+SELECT TO_TIMESTAMP('2018-05-15 00:01:02');
```

Comment:
Can you check this https://docs.snowflake.com/en/sql-reference/functions/to_time, time only expects, parsable time format.

Reply:
This should be the output.

```
SELECT DATE_FORMAT(TO_TIMESTAMP('00:01:02'), 'HH:mm:ss');